server/models/Resolvers.ts

```diff
@@ -1,41 +1,73 @@
-export const resolvers = {
+import User from './User';
+
+const bcrypt = require('bcryptjs');
+
+type QueryUserArgs = {
+    email: string;
+};
+
+type CreateUserArgs = {
+    input: { email: string; password: string };
+};
+
+type EmailArg = {
+    email: string;
+};
+
+type User = {
```

Comment:
This is identical to the the `User` import. No reason to redefine the type here. Depending on where we're using these types, it may be worth creating a `@types` directory which contains all of our `.d.ts` files for the project.

Reply:
So far, we're only using these types in this file, but if we ever need to utilize GraphQL Resolver context or add functionality in other areas then we can make a types file.